playbooks/provision.yml

```diff
@@ -54,6 +54,8 @@
       tags: app
 
     - role: dbserver # Set up database server and user for the app.
+      become: yes
+      become_user: root
```

Comment:
Yes! It depends on the context though, it can change. I encountered some weird issues with fatal permissions errors when testing this PR, and this fixed them. I didn't want to spend too much time investigating it, as it doesn't hurt.